### PR TITLE
<Carousel/> - Fix indicators color and margin-top

### PR DESCRIPTION
**Pull request by bamai**
### 🔦 Summary
Fix indicators color and margin-top: https://jira.wixpress.com/browse/DSM-132
...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/Carousel/Carousel.js [hunk 1]
```diff
@@ -146,11 +146,12 @@ class Carousel extends React.Component {
         aspectRatio={Proportion.PREDEFINED_RATIOS.landscape}
       >
         <div
+          style={{ height: '100%' }}
           data-hook={dataHooks.imagesContainer}
           data-is-loading={this._isLoading()}
         >
           <img
-            src={image.src}
+            {...image}
```
Comment (TalZa1810) [inline src/Carousel/Carousel.js:154]: Can you clarify what's the purpose of the change?
Do we test this change?

Comment (TalZa1810) [reply src/Carousel/Carousel.js:154]: @EgoziE do you think that we should add a test for this change? and if so- a visual test or unit test? WDYT?

Comment (EgoziE) [reply src/Carousel/Carousel.js:154]: Right now this image variable is just an object containing `src`.
It should be of type `ImgHTMLAttributes` and be spread over the `` element.

1. Need to change the d.ts accordingly (I'm not sure what is the right way to write it, please consult Moshe/Guilad/Google)
2. No need to test this specific change, it's just for adding native props like `alt` etc.

Comment (mosheKerbel) [reply src/Carousel/Carousel.js:154]: @maia-b 
```jsx
images?: CarouselImage[];
export type CarouselImage = Partial<HTMLImageElement> & { src: string };
```
What happens here?
You are telling TS that the item **must** contain an `src` prop, but it **can** also contain, part of the props from `HTMLImageElement`
`Partial<x>` makes all props from x to be optional.
Make sure to test it in the `carousel.tsx` types test.

`HTMLImageElement` exists out of the box, meaning you get it automatically from the typescript environment, you do not have to import it in your d.ts.
